src/python/pants/backend/python/lint/python_fmt.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
GitHub is showing this as a new file, but it's very similar to `python_formatter.py` down below. The body of the rule is almost identical. The dataclasses are all different.